extensions/vertx-http/runtime/src/main/java/io/quarkus/devui/runtime/ReportAnIssueJsonRPCService.java

```diff
@@ -0,0 +1,98 @@
+package io.quarkus.devui.runtime;
+
+import java.io.File;
+import java.io.UnsupportedEncodingException;
+import java.net.URL;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+import java.util.Objects;
+
+import jakarta.enterprise.context.ApplicationScoped;
+
+import org.zeroturnaround.exec.ProcessExecutor;
+
+import io.smallrye.common.os.OS;
+import io.vertx.core.json.JsonObject;
+
+@ApplicationScoped
+public class ReportAnIssueJsonRPCService {
+
+    public JsonObject reportBug() {
+        URLBuilder urlBuilder = new URLBuilder(
+                "https://github.com/quarkusio/quarkus/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml");
+        gatherInfo(urlBuilder);
+        return new JsonObject(
+                Map.of("url", urlBuilder.toString()));
+    }
+
+    String run(String... command) {
+        try {
+            String output = new ProcessExecutor().command(command).readOutput(true).execute().outputUTF8();
+            return output;
+        } catch (Exception e) {
+            return "";
+        }
+    }
+
+    void gatherInfo(URLBuilder builder) {
+        builder.addQueryParameter("java_version", System.getProperty("java.version"))
+                .addQueryParameter("quarkus_version",
+                        Objects.toString(getClass().getPackage().getImplementationVersion(), "999-SNAPSHOT"));
+
+        if (OS.WINDOWS.isCurrent()) {
```

Comment:
shouldn't these calls be done wether the project has a build.gradle/build.kts and/or  pom.xml file?

and what if they don't use mvnw/gradlew wrappers? should have fallback to call mvn/gradle in path?